SPECS/mstflint/mstflint.spec

```diff
@@ -0,0 +1,298 @@
+%global __remake_config 1
```

Comment:
If that's hard-coded, we might as well remove it and update the spec accordingly.

Reply:
Done. Thanks!